### ISSUE TITLE
DaaS configuration not included in MetadataAPIConfig.properties by default

### ISSUE THREAD

**Issue by ligadata-william** [LigaData/Kamanja#1560]
Configuration such as this:
```
DaasConfig={"requestTopic": {"kafkaHost": "localhost:9092", "topicName": "testin_1"}, "responseTopic": {"kafkaHost": "localhost:9092", "topicName": "testout_1", "numberOfThreads": "1"}, "inputTopic": {"kafkaHost": "localhost:9092", "topicName": "inputtopic"}}
```
Needs to be added to MetadataAPIConfig.properties by default. It can, at the least, be commented out.

**Comment by ligadata-yousef** [comment]
@ligadata-william 

I am working on this.

**Comment by ligadata-yousef** [comment]
@ligadata-william

the fix is in this branch 1.5.3-phoenix-daas-api-input-md-move-update

**Comment by ligadata-william** [comment]
Verified as fixed. Merged into 1.5.3-phoenix-daas-api-input-md-move
